go.mod

```
@@ -39,6 +39,7 @@ require (
 	github.com/facebookgo/subset v0.0.0-20150612182917-8dac2c3c4870 // indirect
 	github.com/gliderlabs/ssh v0.3.1
 	github.com/glycerine/go-unsnap-stream v0.0.0-20190901134440-81cf024a9e0a // indirect
+	github.com/go-chi/chi v4.1.2+incompatible
```

Just saw that go mod based go-chi v1.5.0 has just been released, we should use it
```suggestion
	github.com/go-chi/chi v1.5.0
```